proxy/pom.xml

```diff
@@ -598,6 +603,16 @@
       <artifactId>snakeyaml</artifactId>
       <version>2.0</version>
     </dependency>
+    <dependency>
+      <groupId>net.jcip</groupId>
```

Comment:
why?

Reply:
It was automatically added to recognize **NotThreadSafe** annotation.  Removed, thanks